test/integration/bad/reserved/stanc.expected

```diff
@@ -380,8 +380,19 @@ Semantic error in 'struct.stan', line 2, column 7 to column 13:
 
 Identifier 'struct' clashes with reserved keyword.
 [exit 1]
-  $ ../../../../../install/default/bin/stanc sum_to_zero.stan
-Syntax error in 'sum_to_zero.stan', line 2, column 7 to column 25, parsing error:
+  $ ../../../../../install/default/bin/stanc sum_to_zero_matrix.stan
+Syntax error in 'sum_to_zero_matrix.stan', line 2, column 7 to column 25, parsing error:
+   -------------------------------------------------
+     1:  data {
+     2:    real sum_to_zero_matrix;
+                ^
+     3:  }
+   -------------------------------------------------
```

Comment:
Did a quick check on github and luckily I see no code that uses `matrix[*, *] sum_to_zero_matrix;`

https://github.com/search?q=sum_to_zero_matrix+language%3AStan+&type=code